README.md

```diff
@@ -9,34 +9,20 @@ This repository contains a dark theme for Jenkins.
 
 ## Usage
 
-To install this theme use the [Simple Theme Plugin](https://plugins.jenkins.io/simple-theme-plugin/).
```

Comment:
It would be great to keep a "Manual setup without a plugin" section in a separate page. It is likely to be a valid use-case for some instances where plugin management is restricted by IT policies